tests/unit/query_streams.cpp

```diff
@@ -183,6 +197,13 @@ TEST_F(StreamsTest, RestoreStreams) {
       stream_info.common_info.batch_interval = std::chrono::milliseconds((i + 1) * 10);
       stream_info.common_info.batch_size = 1000 + i;
       stream_check_data.owner = std::string{"owner"} + iteration_postfix;
+
+      if (i == 1 || i == 3) {
+        stream_info.configs.emplace(std::string{"sasl.username"}, std::string{"username"} + iteration_postfix);
+      }
+      if (i == 2 || i == 3) {
+        stream_info.credentials.emplace(std::string{"sasl.password"}, std::string{"password"} + iteration_postfix);
+      }
```

Comment:
Could you elaborate a bit on these conditions?

Reply:
Yes, those are just "random" numbers, to make the CONFIGS and CREDENTIALS map vary between consumers. Basically:
- 0 means no config, no credential
- 1 means only config
- 2 means only credential
- 3 means both configuration and credential is set

Reply:
Maybe put that into comments so it can be easily understood by glossing over.